### PR TITLE
ci: clean up auto changelog on releases

### PR DESCRIPTION
**Pull request by henryiii**
I noticed on the last release that the auto generated changelog included some bot entries. Those can be cleaned out with a config file like this.

### CODE REVIEW

File: .github/release.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+changelog:
+  exclude:
+    authors:
+      - dependabot
+      - pre-commit-ci
```
Comment (Gitznik) [inline .github/release.yml:5]: I reckon we can also add the `github-actions` user

```suggestion
      - pre-commit-ci
      - github-actions
```

**Review by Gitznik — CHANGES_REQUESTED** [review]
good call :+1:

**Comment by henryiii** [comment]
I wonder if the [bot] is part of the name if if it's possible to match on that?

**Review by Gitznik — APPROVED** [review]

**Comment by Gitznik** [comment]
> I wonder if the [bot] is part of the name if if it's possible to match on that?

Does not look like it from what I can see in the docs. I think this is fine :+1:

**Comment by chrysle** [comment]
Hmm, I think this might be needed; similar problem at https://github.com/jazzband/pip-tools/pull/1976#issuecomment-1707321760, but we didn't come to test it.

**Comment by henryiii** [comment]
I've been using this as is in a bunch of repos, and it works, you don't need the trailing `[bot]` for sure; it would be nice though if you could trigger on it.

**Pull request merged**